packages/@react-spectrum/overlays/src/Tray.tsx

```diff
@@ -96,22 +88,25 @@ let TrayWrapper = forwardRef(function (props: TrayWrapperProps, ref: RefObject<H
       'spectrum-Tray',
       'react-spectrum-Tray'
     ),
-    otherProps.className
+    styleProps.className
   );
 
-  let domProps = mergeProps(otherProps, overlayProps);
-
   return (
-    <div className={wrapperClassName} style={wrapperStyle}>
-      <div
-        {...domProps}
-        {...modalProps}
-        className={className}
-        ref={ref}
-        data-testid="tray">
-        {children}
+    <>
+      <Underlay {...underlayProps} isOpen={isOpen} />
+      <div className={wrapperClassName} style={wrapperStyle}>
+        <div
+          {...styleProps}
+          {...modalProps}
```

Comment:
possibly missed?
```
style={{
  ...styleProps.style,
  ...modalProps.style
}}
```

Reply:
I guess, but modalProps never has style - it comes from the aria hook. 🤷